tqdm/_tqdm.py

```diff
@@ -561,7 +581,7 @@ def __iter__(self):
                     if delta_t >= mininterval:
                         elapsed = cur_t - start_t
                         # EMA (not just overall average)
-                        if smoothing:  # and delta_it
+                        if smoothing and delta_t:
```

Comment:
Why are you restoring `delta_t`? Is it safe? I remember we had some issues in the past with `delta_it` (but don't remember the details - should do a git blame?)...